contrib/google.golang.org/grpc/option.go

```diff
@@ -148,6 +149,21 @@ func WithIgnoredMethods(ms ...string) Option {
 	}
 }
 
+// WithIgnoredService specifies service to be ignored by the server side interceptor.
+// When an incoming request's full method is in ms, no spans will be created.
+func WithIgnoredService(s grpc.ServiceDesc) Option {
```

Comment:
I am not a fan of having multiple ways to do the same thing. Why not simply implement this in your own program?
```go
func serviceDescPaths(s grpc.ServiceDesc) []string {
	var paths []string
	for _, m := range s.Methods {
		paths = append(paths, fmt.Sprintf("/%s/%s", s.ServiceName, m.MethodName))
	}
	for _, m := range s.Streams {
		paths = append(paths, fmt.Sprintf("/%s/%s", s.ServiceName, m.MethodName))
	}
	return paths
}
```
... and then use the option:
```go
WithIgnoredPaths(serviceDescPaths(_ServiceDesc))
```

No need to add this into the library.

Reply:
Thank you for your review. 
I thought constructing full methods from `grpc.ServiceDesc` is a repetitive task, so I believe my pr can solve it. 
But, I can also agree with your opinion too.